packages/worker/src/worker.ts

```diff
@@ -403,6 +403,18 @@ export class Worker {
   public static async create(options: WorkerOptions): Promise<Worker> {
     const nativeWorkerCtor: WorkerConstructor = this.nativeWorkerCtor;
     const compiledOptions = compileWorkerOptions(addDefaultWorkerOptions(options));
+    Runtime.instance().logger.info(`Worker options:`, {
```

Comment:
@bergundy You mentioned a concern about using `Runtime.instance().logger`, which might prevent user form installing its own logger... Not sure about it, but if you still think this is a valid concern, then we'd have to also fix the `getOrCreateBundle(...)` call just bellow.

Reply:
Here it's not an issue, I'd just avoid using the logger before a call to create a worker because that would instantiate a runtime with the default options.

Reply:
```suggestion
    Runtime.instance().logger.info('Creating worker', { options: {
```